velox/common/base/SpillConfig.cpp

```diff
@@ -34,6 +34,8 @@ SpillConfig::SpillConfig(
     uint64_t _maxSpillRunRows,
     uint64_t _writerFlushThresholdSize,
     const std::string& _compressionKind,
+    bool _enablePrefixSort,
+    PrefixSortConfig _prefixSortConfig,
```

Comment:
Can we pass std::optional<PrefixSortConfig>? We still need a query config but just create an optional PrefixConfig and pass to SpillConfig? Thanks!

Reply:
Do you mean drop `_enablePrefixSort` in SpillConfig.cpp and use std::optional<PrefixConfig> to enable prefix sort or not?